db/migrate/20230112140053_add_review_status_review_reason_and_timestamps_to_crime_applications.rb

```diff
@@ -0,0 +1,15 @@
+class AddReviewStatusReviewReasonAndTimestampsToCrimeApplications < ActiveRecord::Migration[7.0]
+  def change
+    add_column :crime_applications, :review_status, :string
+    add_column :crime_applications, :review_received_at, :timestamp
+    rename_column :crime_applications, :completed_at, :review_completed_at
+    add_column :crime_applications, :return_reason, :jsonb, default: {}
```

Comment:
I know this is probably anticipating the (unknown) future, but this might be a `review_details` jsonb attribute that can handle the return reason and additional attributes and information if needed?

The other attributes could also form part of that object? (the timestamps mainly) although I think you need them in the top level for indexing etc. purposes right?